common/locales/de.json

```diff
@@ -254,6 +264,7 @@
     "reject": "Ablehnen",
     "report": "Bericht",
     "required": "erforderlich",
+    "retentionPolicy": "automatisches Löschen - in Tagen",
```

Comment:
Kannst du die Texte und Labels mit Großbuchstaben beginnen.
Und ich fände es passender wenn du die einzelnen Labels.xy für die settings "wörtlicher“ übersetzt. Die Variablennamen sind ja schon sprechend, dann sollte das auch in der Anzeige so benannt werden